### ISSUE TITLE
Improve JS Api 

### ISSUE THREAD

**Issue by TimoGlastra** [hyperledger/anoncreds-rs#103]
As @genaris has highlighted in # the API for parsing objects could be improved.

currently you have to do the following to load a credential definition:

```
rsCredentialDefinitions[credDefId] = CredentialDefinition.load(JSON.stringify(credentialDefinitions[credDefId]))
```

It would be nice if `load` took a json object instead of a string. We also discussed that e.g. `Presentation.create` could just take json objects as input, and do the loading in the `Presentation` class so the user doesn't need to care about the object handles etc..

I think we should make the API just json objects wherever we can

**Comment by genaris** [comment]
I think it will be useful to implement these improvements before https://github.com/hyperledger/aries-framework-javascript/issues/1331, as otherwise users will need to care about freeing a lot object handles that we most likely will not be useful other than being inputs as some classes like `Presentation`.

**Comment by blu3beri** [comment]
@genaris did #129 and #130 fix this?

**Comment by genaris** [comment]
Yes, mostly. For some few cases the handle clearing by the app will still be needed (for instance the one of the resulting presentation in the example)
